internal/xds/rbac/rbac_engine.go

```diff
@@ -238,3 +272,39 @@ type rpcData struct {
 	// handshake.
 	certs []*x509.Certificate
 }
+
+func (e *engine) doAuditLogging(rpcData *rpcData, rule string, authorized bool) {
+	// In the RBAC world, we need to have a SPIFFE ID as the principal for this
+	// to be meaningful
+	principal := ""
+	if len(rpcData.certs) != 0 && len(rpcData.certs[0].URIs) != 0 &&
+		rpcData.certs[0].URIs[0].Scheme == "spiffe" {
```

Comment:
```suggestion
	if len(rpcData.certs) != 0 && len(rpcData.certs[0].URIs) != 0 && rpcData.certs[0].URIs[0].Scheme == "spiffe" {
```

See [Indentation Confusion](go/go-style/decisions#indentation-confusion)
> Avoid introducing a line break if it would align the rest of the line with an indented code block.